server/datastore/mysql/migrations/tables/20240208095115_AddVulnHostCountsTable.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240208095115, Down_20240208095115)
+}
+
+func Up_20240208095115(tx *sql.Tx) error {
+	createStmt := `
+		CREATE TABLE IF NOT EXISTS vulnerability_host_counts (
+			cve VARCHAR(20) NOT NULL,
+			team_id int(10) UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
Just want to make sure I understand this.
For each CVE, for each team_id you will count the number of hosts?

so each team will have its own line?

Reply:
exactly, we use that same pattern in existing host count tables (ie.  `software_host_counts`